packaging/windows/clisdk/bundle.wxs

```diff
@@ -14,15 +14,41 @@
       <bal:WixStandardBootstrapperApplication
         LicenseFile="dummyeula.rtf"
         ShowFilesInUse="yes"
-        ShowVersion="yes"
-        />
+        ShowVersion="yes" />
 
       <PayloadGroupRef Id="DotnetCoreBAPayloads" />
     </BootstrapperApplicationRef>
 
     <swid:Tag Regid="microsoft.com" InstallPath="[DOTNETHOME]" />
 
-    <Variable Name="DOTNETHOME" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="no" />
+    <util:RegistrySearch Id="CheckSDKInstallLocation"
+            Variable="SDKInstallLocationExists"
+            Result="exists"
+            Root="HKLM"
+            Key="SOFTWARE\dotnet\Setup\InstalledVersions\$(var.Platform)\sdk"
+            Value="InstallLocation" />
+    <util:RegistrySearch Id="SDKInstallLocation"
+            After="CheckSDKInstallLocation"
+            Condition="SDKInstallLocationExists"
+            Variable="installpath"
+            Result="value"
+            Root="HKLM"
+            Key="SOFTWARE\dotnet\Setup\InstalledVersions\$(var.Platform)\sdk"
+            Value="InstallLocation" />
+
+    <util:FileSearch Id="DotnetExeSearch"
+            Variable="DotnetExeExists"
+            Condition="NOT SDKInstallLocationExists"
+            Result="exists"
+            Path="[ProgramFiles6432Folder]dotnet\dotnet.exe"/>
+    <util:DirectorySearch Id="DotnetExeLocation"
+            After="DotnetExeSearch"
+            Condition="DotnetExeExists"
+            Variable="installpath"
+            Path="[ProgramFiles6432Folder]dotnet"/>
+
+    <Variable Name="installpath" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="yes" />
+    <Variable Name="DOTNETHOME" Type="string" Value="[installpath]" bal:Overridable="yes" />
```

Comment:
Is there a reason we have different variables here?  Are they supposed to be set to the same value?

Reply:
installpath I believe is the default, built-in WiX parameter that bundles support (if you run form the commandline with /? it will list 'installpath' as a parameter)

Reply:
Hmm, Id' expect DOTNETHOME to get the default [installpath] value. So if you run the bundle with: /installpath=C:\Foo DOTNETHOME=C:\Bar what happens?

Reply:
"installpath" is a command-line parameter for the SDK bundle that we are now using
"DOTNETHOME" is the familiar install location - our partners teams know about this MSIProperty and support it.

Reply:
Standardizing on "DOTNETHOME"; removed "installpath"